### PR TITLE
Fixes #7647

### LINKED ISSUE #7647: Non-lethal guns allow suicide, rounds are not used
Committing suicide with a gun is possible regardless of whether that gun is capable of inflicting lethal damage, this includes practice and lasertag lasers. Have not checked with floral rays.

Additionally, committing suicide via firearm does not expend a cartridge or unit of charge.

### PR DESCRIPTION
**Pull request by dannno**
fixes #7647
Guns now spend ammunition when you shoot your stupid face off
they also have a much better and more natural suicide message
also you yell BANG when you kill yourself with a banana now

### CODE REVIEW

**Comment by Lumipharon** [comment]
What happens if you suicide with an empty gun?

**Comment by Deantwo** [comment]
I have never used suicide before, so correct me if I am wrong.
But can't you be stopped? Say someone disarms you right after the first message?

So shouldn't there be a check to see if your still holding the gun/banana?

**Comment by dannno** [comment]
@lumipharon you pretend to suicide (you still die)

**Comment by demonfiren** [comment]
I assume if you suicide with an empty gun or a banana it's the sheer stupidity that kills you.
(Also, do mimes yell "Bang!" as well?)

File: code/modules/hydroponics/grown.dm [hunk 1]
```diff
@@ -481,7 +481,9 @@
 /obj/item/weapon/reagent_containers/food/snacks/grown/banana/suicide_act(mob/user)
 	user.visible_message("<span class='suicide'>[user] is aiming the [src.name] at themself! It looks like \he's trying to commit suicide.</span>")
 	playsound(loc, 'sound/items/bikehorn.ogg', 50, 1, -1)
-	sleep(15)
+	sleep(25)
+	user.say("BANG!")
```
Comment (Miauw62) [inline code/modules/hydroponics/grown.dm]: you'll have to carry that weight

**Comment by dannno** [comment]
Mimes don't yell anything ever
I'll fix the disarm gun thing

**Comment by dannno** [comment]
someone merge this shit

**Comment by dannno** [comment]
its fix merge ittttttttttttt

**Comment by demonfiren** [comment]
Paprika has to get all the stun-nerfs through first. Honk.

**Pull request merged**